.changeset/funny-trains-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-react': minor
+'@backstage/plugin-catalog': minor
+---
+
+Added namespace filter and column in default catalog table
```

Comment:
Many adopters have [customized the catalog](https://backstage.io/docs/features/software-catalog/catalog-customization/), so it would be nice to add some instructions here like:

Added an entity namespace filter and column on the default catalog page.

If you have a custom version of the catalog page, you can add this filter in your CatalogPage code:

```diff
          <CatalogFilterLayout>
            <CatalogFilterLayout.Filters>
              <EntityTypePicker />
              <UserListPicker initialFilter={initiallySelectedFilter} />
              <EntityTagPicker />
+              <EntityNamespacePicker />
            </CatalogFilterLayout.Filters>
            <CatalogFilterLayout.Content>
              <CatalogTable columns={columns} actions={actions} />
            </CatalogFilterLayout.Content>
          </CatalogFilterLayout>
```

The namespace column can be added using `createNamespaceColumn()`; this is only needed if you customized the columns for `CatalogTable`.